uqcsbot/base.py

```diff
@@ -74,12 +83,25 @@ def _handle_command(self, event_data: dict) -> None:
 
     def on_command(self, command_name: str):
         def decorator(fn):
-            if not asyncio.iscoroutinefunction(fn):
```

Comment:
Is on_command being deprecated? I see this code got moved to `register_handler` instead.

Reply:
`on_command` is specific to the `Command` wrapper, and I think having such a wrapper is a really good idea. `on` (which uses `register_handler`), however, can be used to subscribe to any events whatsoever. So no, not deprecated, this is just a more specific API

Reply:
How do non-async functions get called now? Since you are no longer wrapping them with run_async.

Reply:
Rip, was intending to replace this differently but evidently I didn't. Code review catches bugs yet again!

Reply:
Fixed